src/stories/packages/components/SpinnerOverlay.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { SpinnerOverlay, SpinnerOverlayProps } from '@terra-ui-packages/components';
+import React from 'react';
+
+const overlayModes: Array<SpinnerOverlayProps['mode']> = [
+  'Default',
+  'Top',
+  'Transparent',
+  'Fixed',
+  'Absolute',
+  'FullScreen',
+];
+
+const placeholderText =
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et\n' +
+  ' dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip\n' +
+  ' ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore\n' +
+  ' eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia\n' +
+  ' deserunt mollit anim id est laborum.';
+
+const meta: Meta<typeof SpinnerOverlay> = {
+  title: 'Packages/Components/SpinnerOverlay',
+  component: SpinnerOverlay,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+  argTypes: {
+    mode: {
+      control: 'select',
+      options: overlayModes,
+      description: 'layout mode',
+      table: {
+        defaultValue: { summary: 'Default' },
+      },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof SpinnerOverlay>;
+
+export const Overlay: Story = {
+  args: {},
+  parameters: {
+    design: {
+      type: 'figma',
```

Comment:
I would recommend leaving out the Figma section (perhaps just comment out the whole thing) because it's confusing to see the button Figma component in the spinner documentation.

Reply:
disabled.